src/main/java/graphql/analysis/MaxQueryComplexityInstrumentation.java

```diff
@@ -88,11 +89,11 @@ protected AbortExecutionException mkAbortException(int totalComplexity, int maxC
 
     QueryTraversal newQueryTraversal(InstrumentationValidationParameters parameters) {
         return QueryTraversal.newQueryTraversal()
-                .schema(parameters.getSchema())
-                .document(parameters.getDocument())
-                .operationName(parameters.getOperation())
-                .variables(parameters.getVariables())
-                .build();
+                             .schema(parameters.getSchema())
```

Comment:
can you make sure you are using the https://github.com/graphql-java/graphql-java/blob/master/graphql-java-code-style.xml codestyle? thanks